### PR TITLE
Creating a leave and time off section

### PR DESCRIPTION
**Pull request by Camsmitti**

### CODE REVIEW

File: guides/welfare/Leave and Time Off [hunk 1]
```diff
@@ -1,17 +1,19 @@
 Here is a list of the time off types in HiBob and some examples of what they should be booked for :
 
 ## Holiday 
-We have adopted a flexible holiday policy, where each team member is given the flexibility to take whatever holiday they need.  Please see our flexible holiday page here.
+We have adopted a flexible holiday policy, where each team member is given the flexibility to take whatever holiday they need.  
+Please see our flexible holiday page [here](https://github.com/madetech/handbook/blob/main/benefits/flexible_holiday.md)
```
Comment (Camsmitti) [inline guides/welfare/Leave and Time Off:5]: Please see our [flexible holiday page](https://github.com/madetech/handbook/blob/main/benefits/flexible_holiday.md)

File: guides/welfare/Leave and Time Off [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 
 ## Sick
-We recognise that for our team to perform at the highest possible level, they need to be healthy in mind and body. Please see our sick policy page here.
+We recognise that for our team to perform at the highest possible level, they need to be healthy in mind and body. 
+Please see our sick policy page [here](https://github.com/madetech/handbook/blob/main/guides/welfare/sick_leave.md)
```
Comment (Camsmitti) [inline guides/welfare/Leave and Time Off:9]: Please see our [sick policy page](https://github.com/madetech/handbook/blob/main/guides/welfare/sick_leave.md)

File: guides/welfare/Leave and Time Off [hunk 3]
```diff
@@ -1,17 +1,19 @@
...
 
 ## Medical Leave 
 Medical leave should be used when you need some time off for a medical appointment or medical procedure, where you are not able to flex your time to make up any hours.  
 When you book this leave in HiBob, please select the reason for your medical leave so that we can make sure we put any support in place for you. Your line manager will need to approve this leave.
 
 ## Learning Time 
-This is for your personal learning time that can be used flexibly, more information can be found here.
+This is for your personal learning time that can be used flexibly, more information can be found [here](https://github.com/madetech/handbook/blob/main/guides/learning/README.md)
```
Comment (Camsmitti) [inline guides/welfare/Leave and Time Off:16]: This is for your personal learning time that can be used flexibly, more information on [learning time](https://github.com/madetech/handbook/blob/main/guides/learning/README.md)

File: guides/welfare/Leave and Time Off [hunk 4]
```diff
@@ -20,7 +22,8 @@ This may be requested to support extended absences or unexpected scenarios where
 This is discretionary paid time off when dealing with an upsetting or sensitive situation such as a bereavement.  Approval from your line manager is required for this type of leave.
 
 ## Parental Leave 
-This is for longer-term leave such as maternity/paternity and will require approval from your line manager and the People team. More information can be found here.
+This is for longer-term leave such as maternity/paternity and will require approval from your line manager and the People team. 
+More information can be found [here](https://github.com/madetech/handbook/blob/main/guides/welfare/parental_leave.md)
```
Comment (Camsmitti) [inline guides/welfare/Leave and Time Off:26]: More information on [flexible parental leave ](https://github.com/madetech/handbook/blob/main/guides/welfare/parental_leave.md)

**Review by Camsmitti — COMMENTED** [review]
Updates to link names

**Comment by owenblacker** [comment]
@Camsmitti  I think this one is good to merge now too, no?

**Comment by Camsmitti** [comment]
Looks good to me :)



*Cam Smitti **(She / Her)*

Assoc. People Ops & Culture Advisor


<https://www.madetech.com/?utm_source=signaturesatori&utm_medium=email&utm_campaign=general_signature>

www.madetech.com
<https://www.madetech.com/?utm_source=signaturesatori&utm_medium=email&utm_campaign=general_signature>


twitter.com/madetech

linkedin.com/company/madetech <https://www.linkedin.com/company/madetech/>

**Pull request merged**